src/reconciler.tsx

```diff
@@ -129,8 +134,9 @@ export function applyProps(instance: any, newProps: any, oldProps: any = {}, acc
   // Add left-overs as undefined props so they can be removed
   leftOvers.forEach((key) => key !== 'children' && (filteredProps[key] = undefined))
 
-  if (Object.keys(filteredProps).length > 0) {
-    Object.entries(filteredProps).forEach(([key, value]) => {
```

Comment:
Here, we were performing a redundant operation since `Object.entries` is inclusive of `Object.keys`. It is faster to lift `Object.entries(filteredProps)` to a constant and use that for checking length as well.